cap/modules/services/views/cds.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Analysis Preservation Framework.
+# Copyright (C) 2018 CERN.
+#
+# CERN Analysis Preservation Framework is free software; you can redistribute
+# it and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Analysis Preservation Framework is distributed in the hope that it will
+# be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Analysis Preservation Framework; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""CAP CDS service views."""
+
+import requests
+from flask import jsonify, abort
+
+from . import blueprint
+from ..serializers.cds import CDSRecordSchema
+from cap.modules.access.utils import login_required
+from cap.modules.experiments.errors import ExternalAPIException
+
+from dojson.contrib.marc21.utils import create_record
+import xml.dom.minidom as md
+
+CDS_URL = 'https://cds.cern.ch/record/{}/export/xm?ln=en'
+
+
+def _cds(record_id):
+    """Get Indico event by id."""
+    url = CDS_URL.format(record_id)
+    resp = requests.get(url)
```

Comment:
Needs better error handling

For example 
- CDS record 1 / https://cds.cern.ch/record/1/export/xm?ln=en - FAILS because it is deleted

```
<collection xmlns="http://www.loc.gov/MARC21/slim">
<record>
<controlfield tag="001">1</controlfield>
<datafield tag="980" ind1=" " ind2=" ">
<subfield code="c">DELETED</subfield>
</datafield>
</record>
</collection>
```
- CDS record 3000 / https://cds.cern.ch/record/3000/export/xm?ln=en - FAILS needs authentication 
- CDS record 30000 / https://cds.cern.ch/record/30000/export/xm?ln=en - FAILS